pytorch_to_returnn/torch/_C.py

```diff
@@ -122,17 +124,57 @@ class Size(tuple):  # type: Tuple[SizeValue, ...]
 
 class SizeValue(int):
   """
-  We extend this, to store extra information, e.g. such that this reflects the batch-dim.
+  We extend this, to store extra information, e.g. corresponding RETURNN dim tags.
   """
-  is_batch_dim: bool = False
-  merged_dims: list[SizeValue] = []
+  def __new__(cls, x: int, dim_tag: Optional[Dim] = None, merged_dims: Optional[List[SizeValue]] = None,
+              originating_tensor=None):
```

Comment:
Add type info to `originating_tensor`.

Reply:
See above, we cannot import `Tensor` here due to a circular import. Or how should we do it?